packages/fxa-settings/src/components/CardHeader/index.tsx

```diff
@@ -125,7 +125,6 @@ const serviceLogos: {
 // export const POCKET_CLIENTIDS = [
 //   '7377719276ad44ee', // pocket-mobile
 //   '749818d3f2e7857f', // pocket-web
-//   'dcdb5ae7add825d2', // pocket-web
```

Comment:
Is this incorrect?

Reply:
A sanity check would be good here, but I believe this client ID is actually for `123Done` and was included in the `pocket-migration-mixin` for testing.

Reply:
Yep you're right, this one's for 123done local.

We should be addressing some of this over in Settings here: FXA-8290

Since that ticket only mentions Pocket, mind linking to your Monitor IDs + Monitor check over there?

Reply:
Added Monitor client IDs here and as a comment in [FXA-8290](https://mozilla-hub.atlassian.net/browse/FXA-8290)